mathesar/exceptions/exceptions.py

```diff
@@ -0,0 +1,49 @@
+from collections import namedtuple
+from typing import List
+
+from django.utils.encoding import force_str
+from rest_framework import status
+from rest_framework.exceptions import APIException
+
+from mathesar.exceptions.error_codes import ErrorCodes
+
+ExceptionBody = namedtuple('ExceptionBody',
```

Comment:
I think this will be most readable in one of these two options:

```python
ExceptionBody = namedtuple('ExceptionBody', ['code', 'message', 'field', 'details'], defaults=[None, None])
```

or

```python
ExceptionBody = namedtuple(
    'ExceptionBody',
    ['code', 'message', 'field', 'details'],
    defaults=[None, None]
)
```